src/wp-includes/user.php

```diff
@@ -1988,6 +1988,32 @@ function username_exists( $username ) {
 	return apply_filters( 'username_exists', $user_id, $username );
 }
 
+/**
+ * Determines whether the given user_nicename exists.
+ * For more information on this and similar theme functions, check out
+ * the {@link https://developer.wordpress.org/themes/basics/conditional-tags/
+ * Conditional Tags} article in the Theme Developer Handbook.
+ *
+ * @since 6.6.0
+ *
+ * @param string $nicename Nicename.
+ * @param string $login (Optional) User login to compare against.
+ *
+ * @return int|false The user's ID on success, and false on failure.
```

Comment:
```suggestion
 * @param string $nicename Nicename.
 * @param string $login    Optional. User login to compare against.
 * @return int|false The user ID on success, false on failure.
```

Docblock suggestion.